scalafix-rules/src/main/scala-2.12+/scalafix/internal/rule/TPrintImplicits.scala

```diff
@@ -0,0 +1,13 @@
+package scalafix.internal.rule
+
+import java.util.regex.Pattern
+
+import metaconfig.pprint._
+import scalafix.config.CustomMessage
+
+class TPrintImplicits {
+  implicit val tprintPattern: TPrint[List[CustomMessage[Pattern]]] =
```

Comment:
This implicit has actually no effect (neither before or after this PR) as the signature of the param it was targeting changed in https://github.com/scalacenter/scalafix/pull/907/files#diff-fa445c2f97a1bfb229caa4cd2fdc97cb9c243ddac155698a132bcbdb22970910R91. I will follow-up with a PR